Gems/GradientSignal/Code/Include/GradientSignal/Ebuses/GradientTransformRequestBus.h

```diff
@@ -22,16 +23,56 @@ namespace GradientSignal
         static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
         using BusIdType = AZ::EntityId;
 
-        //! allows multiple threads to call shape requests
+        //! allows multiple threads to call gradient transform requests
         using MutexType = AZStd::recursive_mutex;
 
         virtual ~GradientTransformRequests() = default;
 
-        virtual void TransformPositionToUVW(const AZ::Vector3& inPosition, AZ::Vector3& outUVW, bool& wasPointRejected) const = 0;
-        virtual void TransformPositionToUVWNormalized(const AZ::Vector3& inPosition, AZ::Vector3& outUVW, bool& wasPointRejected) const = 0;
-        virtual void GetGradientLocalBounds(AZ::Aabb& bounds) const = 0;
-        virtual void GetGradientEncompassingBounds(AZ::Aabb& bounds) const = 0;
+        //! Get the GradientTransform that's been configured by the bus listener.
+        //! /return the GradientTransform instance that can be used to transform world points into gradient lookup space.
+        virtual const GradientTransform& GetGradientTransform() const = 0;
     };
 
     using GradientTransformRequestBus = AZ::EBus<GradientTransformRequests>;
+
+   /**
+    * Notifies about changes to the GradientTransform configuration
+    */
+    class GradientTransformNotifications
+        : public AZ::EBusTraits
+    {
+    public:
+        ////////////////////////////////////////////////////////////////////////
+        // EBusTraits
+        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        using BusIdType = AZ::EntityId;
+        using MutexType = AZStd::recursive_mutex;
+        ////////////////////////////////////////////////////////////////////////
+
+        //! Notify listeners that the GradientTransform configuration has changed.
+        //! /return the GradientTransform instance that can be used to transform world points into gradient lookup space.
```

Comment:
nit: I'm guessing the / in front of return was unintentional

Reply:
It was in fact intentional, for Doxygen, but looking at it again, it's backwards, it should be \ not /. I'll fix. :)